node/core/bitfield-signing/src/lib.rs

```diff
@@ -274,7 +289,10 @@ impl JobTrait for BitfieldSigningJob {
 		_receiver: mpsc::Receiver<ToJob>,
 		mut sender: mpsc::Sender<FromJob>,
 	) -> Pin<Box<dyn Future<Output = Result<(), Self::Error>> + Send>> {
+		let metrics = metrics.clone();
 		async move {
+			let _timer = metrics.time_run();
```

Comment:
This timer will always include the `JOB_DELAY` which should be prevented.

Reply:
Good point: 92538c71.